apps/ios/Shared/Views/UserSettings/UserProfile.swift

```diff
@@ -84,14 +83,20 @@ struct UserProfile: View {
                 showImagePicker = true
             }
             Button("Choose from library") {
-                imageSource = .photoLibrary
+                imageSource = .gallery
                 showImagePicker = true
             }
         }
         .sheet(isPresented: $showImagePicker) {
-            ImagePicker(source: imageSource, image: $pickedImage, imageUrl: $tmpImageUrl)
+            if imageSource == .gallery {
```

Comment:
better use switch for enum (you won't need Equatable then, it it checks for completeness)